meson.build

```diff
@@ -14,10 +14,51 @@ endif
 
 libzim_dep = dependency('libzim', version : '>=6.1.1', static:static_linkage)
 
-with_writer = get_option('with-zimwriterfs')
-
 compiler = meson.get_compiler('cpp')
 find_library_in_compiler = meson.version().version_compare('>=0.31.0')
 rt_dep = compiler.find_library('rt', required:false)
 
+with_writer = get_option('with-zimwriterfs')
+
+if with_writer and target_machine.system() != 'windows'
```

Comment:
Maybe you could do :
```
with_writer = get_option('with-zimwriterfs') and target_machine.system() != 'windows'

if with_writer
  ...
```

This way you can simply just test `with_writer` when you include the subdirectory zimwriterfs later.

Reply:
Good point, done: 'if with_writer' condition used in both cases.